nengo_loihi/hardware/interface.py

```diff
@@ -427,6 +433,7 @@ def close(self):
 
         if self.nxsdk_board is not None:
             d_func(self.nxsdk_board, b"ZGlzY29ubmVjdA==")
+            self.nxsdk_board = None
```

Comment:
Can we note in the commit message that closing the simulator now gets rid of the `nxsdk_board`? This might affect some of my scripts. (My guess is no other users are accessing `nxsdk_board`, so probably doesn't need a changelog entry.)

Reply:
Sure, can do.

Reply:
Done in 51262580 (until it gets squashed in)